### DISCUSSION TITLE
Allow custom sendmail configuration

### DISCUSSION THREAD

**Discussion by mmorainville** [strapi/strapi#6188] [Legacy Topics]
Hi,

It's not really a feature request, more an enhancement of an already existing feature.
Right now in `strapi-provider-email-sendmail`, sendmail is instantiated using opiniated params:

```
const sendmail = require('sendmail')({
  silent: true,
});
```

However there is a lot of useful options that can be set: https://github.com/guileen/node-sendmail#options

Especially in my case, I'd need to configure the `devPort` to be able to send emails locally (and intercept them with something like [FakeSMTP](http://nilhcem.com/FakeSMTP/)).

This would allow developers to test sending emails locally without setting or relying on a custom freemium tiers service such as Sendgrid.

It could also solve some other issues like this one: https://github.com/strapi/strapi/issues/3804

If you're interested in the feature, I could try and make a PR, just let me know.

Regards,
M.

**Comment by lauriejim** [comment]
Hello! Thank you for this message.
I suggest you check that with @alexandrebodin here or in the Community Slack.
This could be cool to have this feature. And I will appreciate if you do a PR.
thanks

**Comment by petersg83** [comment]
Hi !
This possiblity will be out soon :)
Made by this PR : https://github.com/strapi/strapi/pull/6210

**Reply by mmorainville** [reply]
Wow good job @petersg83, this goes even beyond what I imagined!
